app/models/concerns/audit_extensions.rb

```diff
@@ -99,6 +100,12 @@ def ensure_taxonomies_not_escalated
 
   private
 
+  def log_audit
+    Foreman::Logging.with_fields self.audited_changes do
+      Foreman::Logging.logger('audit').info("Audit #{self.action} event for #{self.auditable_type} with id #{self.auditable_id}")
```

Comment:
I'd try to make the message at little verbose as possible. The "Audit" word is not needed probably, the fact this is being logged into audit logger is enough. Since logs are usually presented with timestamp at the beginning (WHEN), then we only need to present "WHO" and "WHAT".

For better performance, I'd perhaps add `return` if audit logger is disabled or level is being filtered out. If you don't like this, at least use block call to prevent interpolation.

Reply:
removed the Audit word and coverted it to block, it's only evaluated if logger is configured with appropriate level now